backend/openapi.yaml

```diff
@@ -4881,22 +5097,46 @@ components:
           example: 42
         credentials_label:
           type: string
-          example: "ogp-twilio-creds"
+          example: 'ogp-twilio-creds'
         recipient:
           type: string
-          example: "98765432"
+          example: '98765432'
         body:
           type: string
-          example: "Hello world"
-        message_id:
-          type: string
-          example: "SM12345678901234567890123456789012"
+          example: 'Hello world'
         created_at:
           type: string
           format: date-time
         updated_at:
           type: string
           format: date-time
+        accepted_at:
+          type: string
+          format: date-time
```

Comment:
yeah this is the part where it's `date-time` instead of `date-time (ISO 8601)`

think we should make it consistent for the tx email API also

Reply:
They're the same thing, `date-time` is `ISO 8601`